### PR TITLE
feat: new error page design

### PR DESCRIPTION
**Pull request by nathanielwarner**
### 🔗 Linked issue

https://energysage.atlassian.net/browse/CED-1416

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Updates the styling of the error page for ESDS 2.0 to match the Figma.

### 🥼 Testing

Manual testing

#### 🧐 Feedback Requested / Focus Areas

- General

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
Looks good!

Can we:
- Update all text to be sentence case rather than title case (e.g. "Access denied" instead of "Access Denied" - the design is incorrect here)
- Update the docs page to pass in text for the links that matches the design (except also in sentence case)
- Update the docs page to include an example for each supported error code
- Update the docs page with guidance on how to use this in a page layout (e.g. if we're adding the background color to the component, is this component sufficient to take over the entire viewport? or are we expecting them to also set the same background color on the page itself and set their own top padding of ~224px? if the former, we may want to increase the padding around the content to 100% match the design)

File: es-vue-base/src/lib-components/EsErrorPage.vue [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
         <b-row>
             <b-col
                 md="7"
                 class="my-450 mx-auto text-center">
-                <h1 class="font-weight-boldest display-3">
-                    {{ errorType }}
+                <h1 class="font-weight-bolder display-3 text-gray-600">
+                    {{ errorShortMessage }}
                 </h1>
                 <h2
```
Comment (ericdouglaspratt) [inline es-vue-base/src/lib-components/EsErrorPage.vue:10]: This was pre-existing, but semantically, this content should be a `<p>`, not an `<h2>` tag, as it's not a heading for content underneath it - can we update this as well

**Comment by nathanielwarner** [comment]
> we may want to increase the padding around the content to 100% match the design)

@ericdouglaspratt To clarify, should I hardcode the pixel values for this? Using the `py-800` class gets us most of the way there but it's not 100% matching the design.

**Comment by ericdouglaspratt** [comment]
I'd say whatever padding utility class is closest. Doesn't need to be exact.

File: es-vue-base/src/lib-components/EsErrorPage.vue [hunk 2]
```diff
@@ -1,22 +1,17 @@
 <template>
-    <b-container class="my-450">
+    <b-container class="py-450 py-lg-800 bg-gray-50">
         <b-row>
             <b-col
                 md="7"
                 class="my-450 mx-auto text-center">
-                <h1 class="font-weight-boldest display-3">
-                    {{ errorType }}
+                <h1 class="font-weight-bolder display-3 text-gray-600">
```
Comment (ericdouglaspratt) [inline es-vue-base/src/lib-components/EsErrorPage.vue:7]: Now that we have it, can we swap out `font-weight-bolder display-3` for `post1` class?

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
Just one small thing

File: es-vue-base/src/lib-components/EsErrorPage.vue (hunk 2 above)
Comment (nathanielwarner) [reply es-vue-base/src/lib-components/EsErrorPage.vue]: Done!

**Review by ericdouglaspratt — APPROVED** [review]
Nice!

**Pull request merged**